src/hps_align/plot/vertex.py

```diff
@@ -0,0 +1,91 @@
+import os
+import ROOT as r
+from ._plotter import Plotter
+from .index_page import htmlWriter
+
+
+@Plotter.user
+def multi_vtx(p):
```

Comment:
This does not work but if I remember correctly this function also didn't work in my initial version. We need to keep this in mind and fix eventually

Reply:
This is working for me but I am using a 2016 data set and (I believe) a different hps-java steering file from you, so I think its close (at least).